impl/src/main/java/slack/ThreadMessageHandler.java

```diff
@@ -62,7 +62,7 @@ public boolean shouldHandle(String type, JsonObject body) {
                 //No thread found -> create new
                 .onErrorResumeNext(throwable -> {
                     if (throwable instanceof WebException
-                            && NOT_FOUND.equals(((WebException) throwable).getStatus())) {
+                        && NOT_FOUND.equals(((WebException)throwable).getStatus())) {
```

Comment:
This linebreak is still horrible though 😄

Reply:
fixed :)